Update dark mode illustrations

irfano opened the pull request:
<!-- Remember about a good descriptive title. -->

<!-- Id number of the GitHub issue this PR addresses. -->

### Description
<!-- Take the time to write a good summary. Why is it needed? What does it do? When fixing bugs try to avoid just writing “See original issue” – clarify what the problem was and how you’ve fixed it. -->
During the design review, it was commented that the illustrations we use for Jetpack setup flow don't look great in dark theme, so we got new versions that use lighter purple, this PR updates those illustrations. 
I also forgot to update the light mode of empty store picker illustration, and updated it here.

### Steps to reproduce
<!-- Step-by-step testing instructions. For new user flows, consider instead stating the goal of the workflow and see if your PR reviewer can accomplish the workflow without specific steps! -->
Checking the screenshots below should be enough.

#### Reproduce Empty Site Picker
1. Log in to a site using an account that is associated with only one site.
2. After logging in, use another admin account to delete the user from the site.
3. Go to Menu → Site picker

### The tests that have been performed
<!-- To give the reviewer an idea of what could be missed in terms of testing -->
Screens below

### Images/gif
<!-- Include before and after images or gifs when appropriate. -->
|Before|After|
|-|-|
|<img src="https://github.com/user-attachments/assets/b068a665-4fff-49e9-a823-bdb9fbda254c" width=480>|<img src="https://github.com/user-attachments/assets/053f812a-0aa8-41cf-89f3-539729d0fe70" width=480>|
|<img src="https://github.com/user-attachments/assets/f64ecea2-1192-48f5-becf-d165771d101d" width=480>|<img src="https://github.com/user-attachments/assets/262144ab-1314-450c-b24a-185237a34f94" width=480>|
|<img src="https://github.com/user-attachments/assets/2eec3202-7129-4b13-bde2-ecb5703d04b9" width=480>|<img src="https://github.com/user-attachments/assets/70cfa662-40fb-441b-a762-1f19f459d4b3" width=480>|
|<img src="https://github.com/user-attachments/assets/fcb00de7-acde-4ee5-866f-6a3cd4f4c44b" width=480>|<img src="https://github.com/user-attachments/assets/b0468140-02a4-4ab6-91d8-cd641ac67f24" width=480>|

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [ ] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [ ] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

<!-- Pull request guidelines: https://github.com/woocommerce/woocommerce-android/blob/develop/docs/pull-request-guidelines.md -->

irfano committed:
Update img_install_jetpack.png for dark mode

irfano committed:
Update img_connect_jetpack.png for dark mode

irfano committed:
Update img_jetpack_connection_dismissed.png for dark mode

irfano committed:
Update img_site_picker_no_stores.webp.png

JorgeMucientes commented:
Hey @irfano nicely done! I was able to test almost all the cases. However there's one screen I was not able to test:

<img width="300"  src="https://github.com/user-attachments/assets/ba968e5e-f8f6-47b0-8687-9256a2f1daa5"> 

How do you log in with an account without any sites? Because we currently force the user to enter a site url, so I always land on this screens: 

https://github.com/user-attachments/assets/a1e672fc-15b1-4d85-95ab-95c994067a18

(I've deleted that single site that is shown on the list, but still shows there, so the site picker is never empty)

Also I wanted to ask about this illustration from the flow above. Is it correct? Because it feels like it has the "old" look and feel, despite having the new Woo logo there:

<img width="300"  src="https://github.com/user-attachments/assets/52394dfe-1611-40b4-843c-0014299fa6a9"> 

Lastly, wanted to ask about the WooCommerce branding on the login email template. Is it under your radar to report this to get it updated? Because it currently shows the old style: 

<img width="300"  src="https://github.com/user-attachments/assets/f6506510-52db-4de0-b503-fcd0a872c545">

irfano commented:
> How do you log in with an account without any sites? Because we currently force the user to enter a site url, so I always land on this screens:

I removed the user from the site after logging in. I added the instructions to the description.

> Also I wanted to ask about this illustration from the flow above. Is it correct?

Yes, that's correct. We only updated the logo and colors on it. Figma reference: Ntkk0ER4hHt5y9KjEGIiGk-fi-3450_5600

> Lastly, wanted to ask about the WooCommerce branding on the login email template. Is it under your radar to report this to get it updated? Because it currently shows the old style:

It’s expected that it’s currently showing the old logo for now. But I still reported it to ensure that updating it is included in our plans. (Internal link: p1738072483131849-slack-C07E7D0R761)
